jlh/WeatherApp/app/src/test/java/com/e_conomic/weatherapp/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package com.e_conomic.weatherapp;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+    @Test
+    public void addition_isCorrect() throws Exception {
```

Comment:
I might be wrong, but is it not that one is for UI tests, i.e. they require that the app is launched in the emulator, and the other is for unit tests that can executed without launching the emulator? ping @tudorgk